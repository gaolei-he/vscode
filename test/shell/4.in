26
/ba
/c/dc
/d/dcc
/cb
/c/cdd
/d/cda
/abd/cad/dbd
/cb/dcb
/b
/bdc/c/cc
/cd/ba/bb
/a
/b
/d
/bc
/d/bab/a
/cbd/dda/c
/bc/b/c
/b/bc
/c/bba
/dc
/cb
/cd/dca
/abc
/cbd
/bda
18
ls
pwd
ls
ls
pwd
pwd
cd ..
pwd
cd d
pwd
cd .
pwd
cd dcc
pwd
cd .
pwd
ls
pwd
