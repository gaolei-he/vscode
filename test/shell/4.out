a
abc
abd
b
ba
bc
bda
bdc
c
cb
cbd
cd
d
dc
/
a
abc
abd
b
ba
bc
bda
bdc
c
cb
cbd
cd
d
dc
a
abc
abd
b
ba
bc
bda
bdc
c
cb
cbd
cd
d
dc
/
/
/
/d
/d
/d/dcc
/d/dcc
/d/dcc
